{"uid":"ea13fb5b37c723e1","name":"Create a new Order for new Customer","fullName":"TestOrder.TestOrder#test_create_new_user_order_5___frugal_latido_com_np____Test_123____9881012080____abced_Test_5____testuser180_mail_com____2023_05_29____Siliguri____female____Exemplar__male_____Bronze_Mist_Antique____42____Golden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Skin_Tight____Test_002____2023_05_29___100900_","historyId":"24484abe6920133ac6da94ddbbbc9e9b","time":{"start":1685344699541,"stop":1685344731073,"duration":31532},"status":"broken","statusMessage":"TypeError: catching classes that do not inherit from BaseException is not allowed","statusTrace":"self = <Pages.Order.Customer.CreateNewUser.CreateNewUser object at 0x0000024E0E43F4C0>\naddress = 'Siliguri'\n\n    def input_address(self, address):\n        try:\n            user_address = self.wait.until(EC.visibility_of_element_located(self.InputAddress))\n            user_address.send_keys(address)\n            time.sleep(1)\n            user_address.send_keys(Keys.SPACE)\n>           self.wait.until(EC.presence_of_element_located((By.CLASS_NAME, \"pac-container\")))\n\n..\\Pages\\Order\\Customer\\CreateNewUser.py:65: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"cee731ca93b3310672ed1631ef254816\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x0000024E0E49A320>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       Backtrace:\nE       \tGetHandleVerifier [0x008E8893+48451]\nE       \t(No symbol) [0x0087B8A1]\nE       \t(No symbol) [0x00785058]\nE       \t(No symbol) [0x007B0467]\nE       \t(No symbol) [0x007B069B]\nE       \t(No symbol) [0x007DDD92]\nE       \t(No symbol) [0x007CA304]\nE       \t(No symbol) [0x007DC482]\nE       \t(No symbol) [0x007CA0B6]\nE       \t(No symbol) [0x007A7E08]\nE       \t(No symbol) [0x007A8F2D]\nE       \tGetHandleVerifier [0x00B48E3A+2540266]\nE       \tGetHandleVerifier [0x00B88959+2801161]\nE       \tGetHandleVerifier [0x00B8295C+2776588]\nE       \tGetHandleVerifier [0x00972280+612144]\nE       \t(No symbol) [0x00884F6C]\nE       \t(No symbol) [0x008811D8]\nE       \t(No symbol) [0x008812BB]\nE       \t(No symbol) [0x00874857]\nE       \tBaseThreadInitThunk [0x76EB7D59+25]\nE       \tRtlInitializeExceptionChain [0x77B5B74B+107]\nE       \tRtlClearBits [0x77B5B6CF+191]\n\n..\\.venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:95: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_new_user_order_5___frugal_latido_com_np____Test_123____98810120...n_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Skin_Tight____Test_002____2023_05_29___100900_>\nusername = 'frugal@latido.com.np', password = 'Test@123'\ncontactNo = '9881012080', name = 'abced Test 5', email = 'testuser180@mail.com'\ndob = '2023-05-29', address = 'Siliguri', gender = 'female'\nproductname = 'Exemplar (male)', leatherProfile = 'Bronze Mist Antique'\nleatherSize = '42', hardware = 'Golden Antique', lining = 'Black'\npolyfill = 'Yes', size = '40', armhole = 5.6, height = 4.4, shoulder = 5\nweight = 46, length = 8, arms = 12, hips = 11, chest = 7, waist = 1, sleeves = 9\nbodytype = 'Skin Tight', remark = 'Test 002', date = '2023-05-29', OTP = 100900\n\n    @allure.title(f\"Create a new Order for new Customer\")\n    @allure.story(\"Create an Order for new Customer \")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forNewCustomer\"))\n    @unpack\n    def test_create_new_user_order(self, username, password, contactNo, name, email, dob, address, gender, productname,\n                                   leatherProfile, leatherSize, hardware, lining, polyfill, size, armhole, height, shoulder, weight,\n                                   length, arms, hips, chest, waist, sleeves, bodytype, remark, date, OTP):\n        self.LoginMethod.nativelogin(self.driver, username, password)\n        try:\n            self.assertEqual(\n>               self.Order.create_new_user_order(contactNo, name, email, dob, address, gender, productname,\n                                                 leatherProfile, leatherSize, hardware, lining, polyfill, size, armhole, height,\n                                                 shoulder, weight, length, arms, hips, chest, waist, sleeves, bodytype, remark,\n                                                 date, OTP),\n                \"Order Succesfully Created\", msg=\"Verification Failed to Create a new Order.\")\n\nTestOrder.py:71: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\executions\\OrdersExecutions\\OrderMethods.py:317: in create_new_user_order\n    self.create_new_user(contactNo, name, email, dob, address)\n..\\executions\\OrdersExecutions\\OrderMethods.py:81: in create_new_user\n    self.fill_form(name, email, dob, address)\n..\\executions\\OrdersExecutions\\OrderMethods.py:71: in fill_form\n    self.createUser.input_address(address)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Pages.Order.Customer.CreateNewUser.CreateNewUser object at 0x0000024E0E43F4C0>\naddress = 'Siliguri'\n\n    def input_address(self, address):\n        try:\n            user_address = self.wait.until(EC.visibility_of_element_located(self.InputAddress))\n            user_address.send_keys(address)\n            time.sleep(1)\n            user_address.send_keys(Keys.SPACE)\n            self.wait.until(EC.presence_of_element_located((By.CLASS_NAME, \"pac-container\")))\n            time.sleep(1)\n            user_address.send_keys(Keys.DOWN)\n            time.sleep(1)\n            self.wait.until(EC.presence_of_element_located((By.CLASS_NAME, \"pac-item\"))).click()\n            time.sleep(1)\n>       except {TimeoutException, NoSuchElementException, Exceptions} as e:\nE       TypeError: catching classes that do not inherit from BaseException is not allowed\n\n..\\Pages\\Order\\Customer\\CreateNewUser.py:71: TypeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestOrder","time":{"start":1685343794978,"stop":1685343794978,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"attachmentsCount":0,"shouldDisplayMessage":false}],"afterStages":[{"name":"_unittest_setUpClass_fixture_TestOrder::0","time":{"start":1685345241109,"stop":1685345241109,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"attachmentsCount":0,"shouldDisplayMessage":false}],"labels":[{"name":"feature","value":"Order List"},{"name":"severity","value":"normal"},{"name":"story","value":"Create an Order for new Customer "},{"name":"suite","value":"TestOrder"},{"name":"subSuite","value":"TestOrder"},{"name":"host","value":"0day"},{"name":"thread","value":"9744-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"TestOrder"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"ea13fb5b37c723e1.json","parameterValues":[]}